libs/shared/console-shared/src/lib/create-general-container/create-general-container.tsx

```diff
@@ -2,14 +2,19 @@ import { useEffect, useState } from 'react'
 import { Controller, useFormContext } from 'react-hook-form'
 import { OrganizationEntity, Value } from '@qovery/shared/interfaces'
 import { InputSelect, InputText, InputTextArea } from '@qovery/shared/ui'
-import { GeneralData } from '../../../../feature/page-application-create-feature/application-creation-flow.interface'
 
 export interface CreateGeneralContainerProps {
   organization?: OrganizationEntity
 }
 
 export function CreateGeneralContainer(props: CreateGeneralContainerProps) {
```

Comment:
if it's a shared-console the name shouldn't Create but only GeneralContainerSettings like git repository